docker/prod/entrypoint.sh

```diff
@@ -44,9 +44,13 @@ if [ "$(id -u)" = '0' ]; then
 	  rm -f $APACHE_PIDFILE || true
 	fi
 
-	if [ ! -z "$ATTACHMENTS_STORAGE_PATH" ]; then
-		mkdir -p "$ATTACHMENTS_STORAGE_PATH"
-		chown -R "$APP_USER:$APP_USER" "$ATTACHMENTS_STORAGE_PATH"
+	# Use ATTACHMENTS_STORAGE_PATH value when OPENPROJECT_ATTACHMENTS__STORAGE__PATH is not set
+	OPENPROJECT_ATTACHMENTS__STORAGE__PATH=${OPENPROJECT_ATTACHMENTS__STORAGE__PATH:-$ATTACHMENTS_STORAGE_PATH}
```

Comment:
I think it should be `export`ed

Reply:
You're right, thanks. I have amended the previous commit.